.mk/docker.mk

```diff
@@ -20,13 +20,19 @@ RUN_CONTAINERS=$(BUILD_CONTAINERS)
 KILL_CONTAINERS=$(BUILD_CONTAINERS)
 LOG_CONTAINERS=$(KILL_CONTAINERS)
 ORG=networkservicemesh
+ifeq ($(shell uname -m), x86_64)
+  VPP_AGENT=ligato/vpp-agent:v1.8.1
+endif
+ifeq ($(shell uname -m), aarch64)
+  VPP_AGENT=ligato/vpp-agent-arm64:dev
```

Comment:
Interesting, why the `:dev` here? Are we going to get reproducible results like this?

Reply:
You are right, no reproducible result.
But the ligato team is now working on version 2 which is more prospective
So I should maybe use https://github.com/ligato/vpp-agent/tree/v2.0.0-beta 
Uses
VPP_REPO_URL=https://github.com/FDio/vpp.git
VPP_COMMIT=a8e3001e68d8f5ea6cf526b131c92f5993597f81 # branch stable/1810

Reply:
Actually the same version of VPP as 1.8.1 ....

Reply:
docker image ligato/vpp-agent-arm64:dev is built on the branch dev of repo https://github.com/ligato/vpp-agent

Reply:
I set the version for arm64 also to v1.8.1